src/site/content/en/progressive-web-apps/manifest-updates/index.md

```diff
@@ -41,19 +41,15 @@ compare it against the local copy.
 
 If select properties in the manifest have changed (see list below), Chrome
 queues the new manifest, and after all windows have been closed, installs it.
-Once installed, all fields from the new manifest (except `name`, `short_name`
-and `start_url`) are updated. If the PWA has been added to the dock on macOS,
-the icon will not update until the PWA is launched from the dock again.
+Once installed, all fields from the new manifest (except `name`, `short_name`,
+`start_url` and `icons`) are updated. 
 
 ### Which properties will trigger an update? {: #cr-desktop-trigger }
 
 * `display` (see below)
-* `icons`
 * `scope`
 * `shortcuts`
 * `theme_color`
-* Pixel changes to primary icon file
-* Pixel changes to splash icon file
 
 {% Aside 'caution' %}
 Changes to `name`, `short_name`, and `start_url` are **not** currently
```

Comment:
We might want to mention `icons` and related icon fields here.

Reply:
done, thank you!